packages/block-library/src/navigation-menu/style.scss

```diff
@@ -61,7 +60,7 @@
 		& > li > ul {
 			margin: 0;
 			position: absolute;
-			background: #fff;
+			//background: #fff;
```

Comment:
This fixes the submenu styling issue for all cases except where the background-color has been unset and defaults to transparent:

<img width="337" alt="Screen Shot 2019-11-08 at 3 22 52 pm" src="https://user-images.githubusercontent.com/8096000/68449604-e6feb680-023b-11ea-80d6-667d8c8ff6ba.png">

We might want to add some inline styles to the submenu to prevent that from happening.